### ISSUE TITLE
Warnings installing debian-9-minimal template

### ISSUE THREAD

**Issue by jpouellet** [QubesOS/qubes-issues#5171]
On a machine which had previously never installed debian-* templates:

```
[user@dom0 ~]$ sudo qubes-dom0-update qubes-template-debian-9-minimal
Using sys-net as UpdateVM to download updates for Dom0; this may take some time...
Last metadata expiration check: 0:20:19 ago on Mon Jul 15 02:46:49 2019.
Dependencies resolved.
================================================================================
 Package                    Arch   Version            Repository           Size
================================================================================
Installing:
 qubes-template-debian-9-minimal
                            noarch 4.0.1-201901271906 qubes-templates-itl 196 M

Transaction Summary
================================================================================
Install  1 Package

Total download size: 196 M
Installed size: 825 M
DNF will only download packages for the transaction.
Downloading Packages:
qubes-template-debian-9-minimal-4.0.1-201901271 9.2 MB/s | 196 MB     00:21    
--------------------------------------------------------------------------------
Total                                           9.2 MB/s | 196 MB     00:21     
Complete!
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Qubes OS Repository for Dom0                                  91 MB/s | 144 kB     00:00    
Dependencies resolved.
=============================================================================================
 Package                           Arch     Version                Repository           Size
=============================================================================================
Installing:
 qubes-template-debian-9-minimal   noarch   4.0.1-201901271906     qubes-dom0-cached   196 M

Transaction Summary
=============================================================================================
Install  1 Package

Total size: 196 M
Installed size: 825 M
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Installing  : qubes-template-debian-9-minimal-4.0.1-201901271906.noarch                1/1 
debian-9-minimal: Importing data
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/qubesadmin/tools/qvm_template_postprocess.py", line 177, in call_postinstall_service
    qubesadmin.config.defaults['shutdown_timeout'])
  File "/usr/lib64/python3.5/asyncio/tasks.py", line 410, in wait_for
    raise futures.TimeoutError()
concurrent.futures._base.TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/bin/qvm-template-postprocess", line 5, in <module>
    sys.exit(main())
  File "/usr/lib/python3.5/site-packages/qubesadmin/tools/qvm_template_postprocess.py", line 301, in main
    loop.run_until_complete(post_install(args))
  File "/usr/lib64/python3.5/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/usr/lib64/python3.5/asyncio/futures.py", line 294, in result
    raise self._exception
  File "/usr/lib64/python3.5/asyncio/tasks.py", line 240, in _step
    result = coro.send(None)
  File "/usr/lib/python3.5/site-packages/qubesadmin/tools/qvm_template_postprocess.py", line 239, in post_install
    yield from call_postinstall_service(vm)
  File "/usr/lib/python3.5/site-packages/qubesadmin/tools/qvm_template_postprocess.py", line 179, in call_postinstall_service
    vm.kill()
  File "/usr/lib/python3.5/site-packages/qubesadmin/vm/__init__.py", line 125, in kill
    self.qubesd_call(self._method_dest, 'admin.vm.Kill')
  File "/usr/lib/python3.5/site-packages/qubesadmin/base.py", line 68, in qubesd_call
    payload_stream)
  File "/usr/lib/python3.5/site-packages/qubesadmin/app.py", line 557, in qubesd_call
    return self._parse_qubesd_response(return_data)
  File "/usr/lib/python3.5/site-packages/qubesadmin/base.py", line 102, in _parse_qubesd_response
    raise exc_class(format_string, *args)
qubesadmin.exc.QubesVMNotStartedError: Domain is powered off: 'debian-9-minimal'
warning: %post(qubes-template-debian-9-minimal-4.0.1-201901271906.noarch) scriptlet failed, exit status 1
Non-fatal POSTIN scriptlet failure in rpm package qubes-template-debian-9-minimal
Non-fatal POSTIN scriptlet failure in rpm package qubes-template-debian-9-minimal
  Verifying   : qubes-template-debian-9-minimal-4.0.1-201901271906.noarch                1/1 

Installed:
  qubes-template-debian-9-minimal.noarch 4.0.1-201901271906                                  

Complete!
[user@dom0 ~]$ 
```

The template installation still seems to have completed fine anyway though.

**Comment by marmarek** [comment]
Killing a domain failed because it was already halted... May mean a race condition, or `qubesd` didn't get an event from libvirt. Can you check if libvirt daemon was restarted recently? Especially if after last `qubesd` restart? (there is missing event handler re-registration after re-connecting to libvirt daemon)

**Comment by jpouellet** [comment]
Indeed, libvirtd was restarted after qubesd, as confirmed by systemctl status libvirtd/qubesd.

**Comment by marmarek** [comment]
This was fixed as part of #5303
